src/main/java/Duke.java

```diff
@@ -2,22 +2,24 @@
 
 public class Duke {
 
-    public static final String EXIT_PROGRAM = "bye";
-    public static final String LIST_TASK = "list";
-    public static final String MARK_DONE = "done";
-    public static final String TODO_TASK = "todo";
-    public static final String DEADLINE_TASK = "deadline";
-    public static final String EVENT_TASK = "event";
+    public static final String COMMAND_EXIT_PROGRAM = "bye";
+    public static final String COMMAND_MARK_DONE = "done";
+    public static final String COMMAND_LIST_TASK = "list";
+    public static final String COMMAND_TODO_TASK = "todo";
+    public static final String COMMAND_DEADLINE_TASK = "deadline";
+    public static final String COMMAND_EVENT_TASK = "event";
+    public static final int COMMAND_INDEX = 0;
```

Comment:
You can have more descriptive names for the commands, such as `COMMAND_ADD_TODO`, `COMMAND_ADD_DEADLINE`, and `COMMAND_ADD_EVENT`.